File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.cpp
class Solution {
public:
    int maxProduct(int n) {
      vector<int>v;
      while(n>0){
        int d=n%10;
        v.push_back(d);
        n=n/10;
      }
      sort(v.begin(),v.end());
      int t=v.size();
      return v[t-1]*v[t-2];
    }
};

File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.java
class Solution {
    public int maxProduct(int n) {
      ArrayList<Integer> l=new ArrayList<>();
      while(n>0){
        int d=n%10;
        l.add(d);
        n=n/10;
      }
      Collections.sort(l);
      int t=l.size();
      return l.get(t-1)*l.get(t-2); 
    }
}